ui/scss/component/_table.scss

```diff
@@ -204,3 +204,29 @@ th {
     margin-left: var(--spacing-m);
   }
 }
+
+.table--publish-preview {
```

Comment:
this should not be a new style

table--file-details and table--publish-preview should probably be unified @seanyesmunt